node/npm_dependencies.ts

```diff
@@ -69,7 +69,7 @@ export const getDependencyTrackerPlugin = (
         return { external: true }
       }
 
-      const isLocalImport = specifier.startsWith(path.sep) || specifier.startsWith('.')
+      const isLocalImport = specifier.startsWith(path.sep) || specifier.startsWith('.') || path.isAbsolute(specifier)
```

Comment:
Fixes an issue where importing an absolute directory on Windows was not being treated as a local import.